js/src/chrome/projects/chrome.projects.js

```diff
@@ -646,23 +641,23 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 			var currentIsChecked = false;
 			var key = event.which ? event.which : event.keyCode;
 
-			if (key !== 40 && key !== 38 && key !== 13) {
-				_.forEach(me._itemsContainers[options.type], function (user, index) {
-					if (user.name.toLowerCase().indexOf(invite_input.value.toLowerCase()) === -1 || _.keys(me._checkedUsers[options.type]).indexOf(user.name) !== -1) {
-						me._itemsContainers[options.type][index].container.style.display = 'none';
-						me._itemsContainers[options.type][index].container.style.backgroundColor = '';
```

Comment:
can you rewrite `me._itemsContainers[options.type]`  to `var item = me._itemsContainers[options.type]` like mentioned before? makes for easier reading.

`var item_index = item[index]` also perhaps